submissions/alyonafuria/html-css_popup/styles.css

```diff
@@ -0,0 +1,174 @@
+ * {
+     box-sizing: border-box;
+     margin: 0;
+     padding: 0;
+     text-decoration: none;
+     list-style-type: none;
+     color: #34495E;
+ }
+ 
+ body {
+     font-family: Arial, Helvetica, sans-serif;
+ }
+ 
+ .navigation {
+     height: 50px;
+     width: 300px;
+     position: fixed;
+     right: 10px;
+     top: 10px;
+     display: flex;
+     justify-content: space-evenly;
+     align-items: center;
+ }
+ 
+ img {
+     max-width: 50px;
+     max-height: 40px;
+ }
+ 
+ .bell {
+     max-width: 25px;
+     max-height: 25px;
+ }
+ 
+ .avatar {
+     border-radius: 50%;
+ }
+ 
+ p {
+     text-align: center;
+ }
+ 
+ .upperButton {
+     background-color: white;
+     display: flex;
+     align-items: center;
+     padding: auto;
+     margin: auto;
+     max-width: 100px;
+     max-height: 30px;
+     font-size: 18px;
+ }
+ 
+ .overlay {
+     display: none;
+     position: fixed;
+     margin: 0;
+     padding: 0;
+     right: 50px;
+     top: 60px;
+     width: 310px;
+     height: 440px;
+     border: solid 1px gainsboro;
+     box-shadow: 0 0 15px rgba(0, 0, 0, 0.2);
+ }
+ 
+ .popup {
+     display: grid;
+     position: relative;
+     margin: auto;
+     padding: auto;
+     height: 100%;
+     width: 100%;
+     grid-template-rows: 100px 100px 100px 100px 38px;
+     grid-template-columns: repeat(3, 100px);
+     grid-auto-rows: 100px;
+     grid-auto-columns: 100px;
+     overflow-y: auto;
+     overflow-x:hidden;
+     grid-auto-flow: row;
+     justify-content: space-between;
+ }
+ 
+ .more {
+     margin: auto;
+     display: none;
+ }
+ 
+ #toggle,
+ #toggle1 {
```

Comment:
Watch the [video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkboxes in the right way